src/styles/variables.scss

```diff
@@ -0,0 +1,13 @@
+$cell-size: 100px;
+$num-of-day-size: 0.3 * $cell-size;
+$hover-color: #ffc0cb;
+$primdry-color: #eee;
```

Comment:
Be careful when naming and do not make typos. ( `$primdry-color` -> `$primary-color` ).